lib/github.js

```diff
@@ -254,6 +254,18 @@ module.exports = {
             await module.exports._runJobAsync({'type': 'create_repo', 'description': 'Creating repository from template'}, job_sequence_id, authn_user.user_id, async (job) => {
                 await module.exports.createRepoFromTemplateAsync(client, options.repo_short_name, config.githubCourseTemplate);
                 job.info(`Created repository ${options.repo_short_name}`);
+
+                /* Find main branch (which is the only branch in the new repo), output of this is array of objects following:
+                   https://docs.github.com/en/rest/reference/repos#list-branches */
+                const branches = await client.repos.listBranches({
+                    owner: config.githubCourseOwner,
+                    repo: options.repo_short_name,
+                });
+                if (branches.length != 1) {
+                    throw `New repo has ${branches.length} branches, expected one.`;
```

Comment:
https://stackoverflow.com/a/11512623
```suggestion
                    throw new Error(`New repo has ${branches.length} branches, expected one.`);
```